wadas/ui/mainwindow.py

```diff
@@ -4,6 +4,7 @@
 import os
 from datetime import timedelta
 from logging.handlers import RotatingFileHandler
+from operator import truediv
```

Comment:
Why this import? I don't find truediv used somewhere else on this PR.

Reply:
it's a rebase

Reply:
@hexfati this shoudl be coming from your latest changes, correct?

Reply:
nope, never seen before now